Composite/Data/Types/PageServices.cs

```diff
@@ -389,6 +409,22 @@ public static IEnumerable<IPage> GetSubChildren(this IPage parentPage)
                         yield return subPage;
                     }
                 }
+
+                if(GlobalSettingsFacade.AllowChildPagesTranslationWithoutParent)
+                {
+                    foreach (Guid emptyPageId in parentPage.GetEmptyChildren())
+                    {
+                        foreach (IPage childPage in GetChildren(emptyPageId))

```

Comment:
Looks like in the sample "parent1 -> empty -> empty -> grandchild1" grandchild1 will not be returned.